### PR TITLE
Add cypress command to verify that required envs variables are set

### PR DESCRIPTION
**Pull request by nixocio**
Add cypress command to verify that required envs variables are set

image

### CODE REVIEW

File: .github/workflows/on-pull-request-run-tests.yml [hunk 1]
```diff
@@ -327,7 +330,7 @@ jobs:
   awx-e2e:
     name: AWX E2E
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```
Comment (jamestalton) [inline .github/workflows/on-pull-request-run-tests.yml:333]: we do not want to bump the timeout up

File: .github/workflows/on-pull-request-run-tests.yml [hunk 2]
```diff
@@ -3,6 +3,9 @@ name: On Pull Request - Run Tests
 on:
   pull_request:
 
+env:
+  IS_GITHUB_ACTION: true
```
Comment (jamestalton) [inline .github/workflows/on-pull-request-run-tests.yml:7]: why would we only check for these variables in github actions?

**Review by jamestalton — CHANGES_REQUESTED** [review]

File: .github/workflows/on-pull-request-run-tests.yml (hunk 2 above)
Comment (nixocio) [reply .github/workflows/on-pull-request-run-tests.yml:7]: Idea was to skip that check in GithubActions.

File: .github/workflows/on-pull-request-run-tests.yml (hunk 1 above)
Comment (nixocio) [reply .github/workflows/on-pull-request-run-tests.yml:333]: It was just a test to verify that the changes were not the ones to cause test failures, I will update.

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**